src/main/java/uk/gov/ons/ctp/response/sample/party/PartyUtil.java

```diff
@@ -22,7 +22,7 @@
    * @return the created Party object
    * @throws Exception unlikely, but indicated something really wrong
```

Comment:
If it no longer throws Exception, please update the Javadoc

Reply:
Done.